src/Adapters/Files/GridFSBucketAdapter.js

```diff
@@ -35,8 +35,6 @@ export class GridFSBucketAdapter extends FilesAdapter {
           .substring(0, 32)
         : null;
     const defaultMongoOptions = {
-      useNewUrlParser: true,
```

Comment:
Why this refactor?

Reply:
yes it fix a mongo warning since it's useless since mongo 4.0 and removed in mongo 7